test/vector-tile.test.js

```diff
@@ -797,59 +797,59 @@ describe('mapnik.VectorTile ', function() {
         var data = fs.readFileSync("./test/data/vector_tile/tile1.vector.pbf");
         vtile.setData(data);
         assert.throws(
-            function() { vtile.getDataSync(null); },
+            function() { vtile.getDataSync(null); }, null,
```

Comment:
just curious: what are those nulls? (I guess node 10 is more picky with assert.throws interface)

Reply:
Node10 changed slightly the API of `assert.throws`. Previously it had the same amount of arguments (function, error, message) but if you passed only (function, message) it was ok. Now I was getting and error due to that (something related to the cast string / error)